babybuddy/Dockerfile

```diff
@@ -49,13 +49,16 @@ RUN \
   rust \
   cargo
 
-RUN \
-  echo "**** installing ****" && \
-  cd /app/babybuddy && \
-  pip3 install -U --no-cache-dir \
-  pip && \
-  pip install -U --ignore-installed -r requirements.txt
+# Define PATH_MAX before including necessary headers
+RUN echo '#define PATH_MAX 4096' >> /usr/include/limits.h
 
+# Install dependencies and pandas
+RUN echo "**** installing ****" && \
+    cd /app/babybuddy && \
+    pip3 install -U --no-cache-dir pip && \
+    pip install --prefer-binary --no-cache-dir pandas && \
```

Comment:
```suggestion
```

Pandas is no longer a dependency. It looks like the latest versions do not support 32bit arch so this is why those arches are failing. Give this a try without this line.